### PR TITLE
chore: Add 2021.1

### PR DESCRIPTION
**Pull request by jeffmaury**
Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### CODE REVIEW

File: .github/workflows/IJ.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IU-2019.3, IU-2020.1, IU-2020.2, IU-2020.3]
+        IJ: [IU-2019.3, IU-2020.1, IU-2020.2, IU-2020.3, IU-2020.1]
```
Comment (lstocchi) [inline .github/workflows/IJ.yml:17]: should be IU-2021.1

File: build.gradle [hunk 2]
```diff
@@ -17,7 +17,7 @@ repositories {
 sourceCompatibility = '1.8'
 targetCompatibility = '1.8'
 
-def versionsMap = ['IU-2019.2':'192.5728.98','IU-2019.3':'193.5233.144','IU-2020.1':'201.6668.99', 'IU-2020.2':'202.6397.93', 'IU-2020.3':'203.5981.152']
+def versionsMap = ['IU-2019.3':'193.5233.144','IU-2020.1':'201.6668.99', 'IU-2020.2':'202.6397.93', 'IU-2020.3':'203.5981.152', 'IU-2021.1':'211.6693.108']
```
Comment (lstocchi) [inline build.gradle:20]: Out of curiosity, what exactly is the reasoning for you to pick the IJ version to use? I can see in the website the newest one is 211.6693.111, why do you use 211.6693.108?

**Review by lstocchi — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (jeffmaury) [reply build.gradle:20]: I'm using this as the source: https://plugins.jetbrains.com/plugin/10485-kubernetes/versions

**Review by lstocchi — APPROVED** [review]
LGTM

**Pull request merged**